src/backend/sev/mod.rs

```diff
@@ -14,8 +14,10 @@ use data::{
     dev_kvm, dev_sev, dev_sev_readable, dev_sev_writable, has_reasonable_memlock_rlimit,
     kvm_version, sev_enabled_in_kernel, CPUIDS,
 };
+
+use kvm_bindings::bindings::kvm_enc_region;
+
 use kvm_ioctls::VmFd;
```

Comment:
Please follow the guidelines https://github.com/enarx/enarx/blob/main/STYLE.md#use-declarations

Reply:
I'm not sure how that does not follow the guide line. It's in its own group.

Reply:
So: can you just say how this is wrong and how it should be and then perhaps we can refine the documentation to make it less likely to make the same mistake?